apps/monitor-client/lib/features/landing/components/BurgerMenu.tsx

```diff
@@ -19,6 +20,10 @@ export function BurgerMenu({ headerRef, featuresRef, stepsRef, roadMapRef, conta
         ref.current?.scrollIntoView({ behavior: 'smooth' });
     };
 
+    const navigate = useNavigate();
+
+    const token = appPersistentStorage.apiToken;
```

Comment:
the token is not a propper name.
maybe something like showAdminFlag?
the same for all others places